Dockerfile

```diff
@@ -27,10 +27,14 @@ RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.
     git \
     python3-file-magic \
     python3-pip \
+    ShellCheck \
     clamav-update && \
     pip3 install --no-cache-dir yq && \
     curl -s -L https://github.com/CycloneDX/sbom-utility/releases/download/v"${sbom_utility_version}"/sbom-utility-v"${sbom_utility_version}"-linux-amd64.tar.gz --output sbom-utility.tar.gz && \
     mkdir sbom-utility && tar -xf sbom-utility.tar.gz -C sbom-utility && rm sbom-utility.tar.gz && \
+    # FIXME: Temporary workaround to install the csmock-plugin-shellcheck without introducing unnecessary dependencies.
+    # Replace this with the installation of csmock-plugin-shellcheck-core once it is stabilized in EPEL.
+    microdnf download csmock-plugin-shellcheck && rpm -ivh --nodeps ./csmock-plugin-shellcheck-*.rpm && rm -f ./csmock-plugin-shellcheck-*.rpm && \
```

Comment:
@kdudka since the package is already in EPEL: https://packages.fedoraproject.org/pkgs/csmock/csmock-plugin-shellcheck-core/

I guess I could get rid of this interim work around now. right?

Reply:
Yes, please.

Reply:
Updated. Since there's the `--setopt=install_weak_deps=0`, `ShellCheck` needs to be installed explicitly.

Reply:
Looks good.  Thanks for the update!